charts/axosyslog-collector/templates/config.yaml

```diff
@@ -20,7 +20,14 @@ data:
 
     log {
 {{- if .Values.config.sources.kubernetes.enabled }}
-      source { kubernetes(); };
+      source { kubernetes(
+  {{- if .Values.config.sources.kubernetes.prefix }}
+        prefix("{{ .Values.config.sources.kubernetes.prefix }}")
+  {{- end }}
+  {{- if .Values.config.sources.kubernetes.keyDelimiter }}
+        key-delimiter("{{ .Values.config.sources.kubernetes.keyDelimiter }}")
+  {{- end }}
+      ); };
```

Comment:
Can we add an example comment to `values.yaml`? Our README is outdated anyway :(